cyences_app_for_splunk/default/data/ui/views/cs_o365_reports.xml

```diff
@@ -53,6 +60,60 @@
       </table>
     </panel>
   </row>
+  <row>
+    <panel>
+      <title>Azure - Current Security Score(In Percentage)</title>
+      <single>
+        <search>
+          <query>`cs_azure_securityscore` sourcetype="graphsecurity:score"  $tkn_host$ | eval percentage = (currentScore/maxScore)*100 | timechart max(percentage) as percentage | streamstats last(percentage) as percentage</query>
+          <earliest>-7d@h</earliest>
+          <latest>now</latest>
+        </search>
+        <option name="numberPrecision">0.0</option>
+        <option name="rangeColors">["0x53a051","0x0877a6","0xf8be34","0xf1813f","0xdc4e41"]</option>
+        <option name="showTrendIndicator">1</option>
+        <option name="unit">%</option>
+        <option name="drilldown">all</option>
+        <drilldown>
+          <set token="tkn_panel_basis_security_score_visible">"True"</set>
+        </drilldown>
+      </single>
+    </panel>
+    <panel>
+      <title>Azure - Current Security Score</title>
+      <single>
+        <search>
+          <query>`cs_azure_securityscore` sourcetype="graphsecurity:score"  $tkn_host$ | head 1 | eval score = currentScore."/".maxScore | fields score</query>
+          <earliest>-7d@h</earliest>
+          <latest>now</latest>
+        </search>
+        <option name="drilldown">none</option>
+        <option name="refresh.display">progressbar</option>
+        <option name="drilldown">all</option>
+        <drilldown>
+          <set token="tkn_panel_basis_security_score_visible">"True"</set>
+        </drilldown>
+      </single>
+    </panel>
+  </row>
+  <row depends="$tkn_panel_basis_security_score_visible$">
+    <panel>
+      <title>Parameter used to calculate Security Score</title>
+      <table id="tbl_security_score">
+        <search>
+          <query>`cs_azure_securityscore` sourcetype="graphsecurity:score" | where $tkn_panel_basis_security_score_visible$=="True"
```

Comment:
may ne set token to empty string (instead of True) and use token at the beginning of the search so search won't execute until usrr drilldown.

Reply:
Bro Search will not execute in this query also until user clicked.

Reply:
yeah i see it.